torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -324,9 +325,38 @@ void ConvertGraphToONNXProto(
   }
 }
 
+bool checkOutputType(Node* n) {
```

Comment:
please be more explicit with the function name, `checkXXX` is too generic and doesn't capture the purpose precisely. (also keep the same naming style with capitalization)
For functions returning boolean, and are not modifying things, consider passing arguments as pointer to constant. And name functions as `IsXXXXX`. 

The code goes straight accessing subblocks. Does the `node` must be an `if` node? Is this method exclusively for if nodes?

Reply:
updated the naming and added asset for if node